libvirt-wrapper.py

```diff
@@ -0,0 +1,91 @@
+import libvirt
+import sys
+import argparse
+
+class LibvirtConn:
+    (ECONN, EFIND, ECREATE, EDELETE, ESTART, ESTOP, EREBOOT) = (1,2,3,4,5,6,7)
+    def __init__(self):
+        try:
+            self.conn = libvirt.open("qemu:///system")
+        except libvirt.libvirtError:
+            raise SystemExit(self.ECONN)
```

Comment:
Using SystemExit in such interfaces is a questionable idea because it's not this class that should decide if it's time to exit.